src/components/dataset-query/dataset-query.scss

```diff
@@ -43,4 +35,75 @@
     margin: 0;
     vertical-align: middle;
   }
+
+  .recommended-dataset {
+    width: 20em;
```

Comment:
Actually, why any `em`s? I'm cool with using them though it would be nice to do it across the board unless we have a really good reason

Reply:
No good reason, force of habit.